### PR TITLE
rename git to Git

### PR DESCRIPTION
**Pull request by schneiderl**
This PR renames "git" to "Git" in all occurrences across tldr (except in command examples and scripts, as it would cause a negative impact).
**Why?**
On the official website, git always starts with an uppercase.

### CODE REVIEW

**Review by bl-ue — APPROVED** [review]

**Review by glenacota — CHANGES_REQUESTED** [review]
Wow, thanks for your great effort, @schneiderl !!
However, there are still a few occurrences of the lowercase git, such as in `CONTRIBUTING.md:L120`, `dvc-diff.md:L10`, `git-instaweb.md:L6`. 

Could you please re-check? Thanks!

**Comment by bl-ue** [comment]
You've got a really sharp eye, @glenacota! 😎

**Comment by schneiderl** [comment]
@glenacota I added those and a few others that were missing - it looks like my regex skills still on vacation.

Thank you for noticing.

**Comment by bl-ue** [comment]
Looks like there's some more @schneiderl:
```
pages.it/common/git-describe.md:18
pages.it/common/git-svn.md:18
pages.zh_TW/common/tldr.md:14
pages.th/common/tldr.md:14
pages.zh/common/tldr.md:14
```

**Comment by schneiderl** [comment]
Nice @bl-ue thank you. 
I've added those.

Thanks for the help.

**Review by bl-ue — APPROVED** [review]

**Review by glenacota — APPROVED** [review]
LGTM, nice one ;)

File: pages/common/chezmoi.md
Comment (navarroaxel) [inline pages/common/chezmoi.md:26]:
```suggestion
- Set chezmoi up on another machine by downloading existing dotfiles from a Git repository:
```

File: pages.es/common/git-submodule.md
Comment (navarroaxel) [inline pages.es/common/git-submodule.md:20]:
```suggestion
`git submodule foreach git pull`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Comment by schneiderl** [comment]
nice catches @navarroaxel

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]

**Comment by waldyrious** [comment]
It's gonna be weird adjusting to seeing Git capitalized, but I concede that it is the logical choice. Go forth and prosper, you have my blessing! 🖖

**Review by waldyrious — APPROVED** [review]

**Review by owenvoke — APPROVED** [review]

**Review by sahilister — APPROVED** [review]

**Comment by sbrl** [comment]
Wow, that's a lot of approvals! I think we can merge this then :P

**Comment by bl-ue** [comment]
It really is!!! But why not just wait a little longer? How about 9 hours, since that's 24 hours after your approval?

**Comment by bl-ue** [comment]
Never mind! You just merged it :D Cool!

**Comment by sbrl** [comment]
Generally speaking PRs should be open for 48 hours before merging @bl-ue, to give people time to review (unless it's a trivial change). But in this case, we have so many approvals it doesn't matter

**Comment by bl-ue** [comment]
Gotcha!

**Comment by bl-ue** [comment]
@sbrl It looks look rebased this, but didn't you intend to squash?

**Review by zlatanvasovic — COMMENTED** [review]
Approved. I thought I had done this a while ago. :open_mouth:

**Comment by bl-ue** [comment]
@zlatanvasovic I saw this just today, I believe you're thinking of #4060 "Github -> GitHub", which I and @schneiderl also did afterwards :smile:

**Comment by sbrl** [comment]
Oops, @bl-ue! I did mean to squash.

**Pull request merged**
